cpp/src/groupby/sort/group_scan.hpp

```diff
@@ -48,12 +48,12 @@ std::unique_ptr<column> sum_scan(column_view const& values,
  * @param values Grouped values to get minimum from
  * @param num_groups Number of groups
  * @param group_labels ID of group that the corresponding value belongs to
- * @param mr Device memory resource used to allocate the returned column's device memory
  * @param stream CUDA stream used for device memory operations and kernel launches.
+ * @param mr Device memory resource used to allocate the returned column's device memory
```

Comment:
Similarly, consistency....
```suggestion
 * @param mr Device memory resource used to allocate the returned column's device memory.
```